src/spaceship.cpp

```diff
@@ -89,6 +89,11 @@ void Spaceship::onFixedUpdate()
                 rope.destroy();
             }
         }
+        if (controls->restart.getDown() && controls->restart.get())
+        {
+            LOG(Debug, "RESET SELF DESTRUCT");
```

Comment:
HELLO SHOUTING MESSAGE, WHICH IS ALSO A BIT UNCLEAR.

Would change it to: LOG(Info, "Initiating self destruct for", controls->index)